test/inductor/test_aot_inductor.py

```diff
@@ -707,16 +704,11 @@ def forward(self, x, weight, bias, scale_a, scale_b):
             dynamic_shapes=dynamic_shapes,
         )
 
-    @unittest.skipIf(
-        not torch.cuda.is_available() or torch.cuda.get_device_capability() < (9, 0),
-        "FP8 is only supported on H100+",
-    )
     @skipIfRocm  # _scaled_mm_out_cuda  is not compiled for ROCm platform
     @skipIfXpu
     def test_fp8_view_of_param(self):
-        # cuda only
-        if self.device != GPU_TYPE:
-            return
+        if self.device == "cuda" and not SM90OrLater:
```

Comment:
It would be nice to keep it in the decorator. Also, isn't there is_fp8_supported helper function?

Reply:
Okay, I will put it in the decorator. And there isn't is_fp8_supported helper function indeed. Let me check and try to define one.

Reply:
@malfet I seem to recall that why I didn't put the check in the decorator.

This test will come in for both CPU and GPU, the information of `self.device` is only inside the test, like https://github.com/pytorch/pytorch/pull/141961/files#diff-6b55ea9919f49e90d28a14bbacc3793c4b3ba932603c141b9452d333c4585642L718. And if the check is in the decorator, we cannot get `self.device`. WDYT?